PedroChaparro/loomies-backend
# ci: Setup database

## Includes

Add an extra step to:

- [x] Inset gyms, zones, loomies, items, and loomballs. 
- [x] Insert random rewards.

## How can I test this? 

Check the pull request checks, you should see the new step and the logs generated by the script: 

![image](https://user-images.githubusercontent.com/62714297/224522985-f911f580-5f84-4d85-8451-0c56e5b70318.png)
